### PR TITLE
suggest a team policy if no household id

### PR DESCRIPTION
**Pull request by hobbitronics**
- added text and a link to create a team policy
Screen Shot 2021-12-03 at 3 20 16 PM

### CODE REVIEW

File: src/components/NoHouseholdIdModal.svelte [hunk 1]
```diff
@@ -49,6 +50,12 @@ const isIdValid = (id: string): boolean => /^[0-9]+$/.test(id)
 
 <Dialog.Alert {open} {buttons} defaultAction="cancel" {title} on:chosen={handleDialog} on:closed={handleDialog}>
   <p>A household ID is required before getting coverage</p>
+  <p>
+    If you would like to pay with a cost center, consider creating a <a
+      class="mdc-theme--primary"
+      href={POLICY_NEW_TEAM}>Team Policy</a
+    >
```
Comment (merchako) [inline src/components/NoHouseholdIdModal.svelte:57]: Add a period at the end of the sentence:
```
    >.
```

File: src/components/NoHouseholdIdModal.svelte [hunk 2]
```diff
@@ -49,6 +50,12 @@ const isIdValid = (id: string): boolean => /^[0-9]+$/.test(id)
 
 <Dialog.Alert {open} {buttons} defaultAction="cancel" {title} on:chosen={handleDialog} on:closed={handleDialog}>
   <p>A household ID is required before getting coverage</p>
```
Comment (merchako) [inline src/components/NoHouseholdIdModal.svelte:52]: Finish sentence with a period.
``` 
<p>A household ID is required before getting coverage</p>.
```

File: src/components/NoHouseholdIdModal.svelte [hunk 3]
```diff
@@ -49,6 +50,12 @@ const isIdValid = (id: string): boolean => /^[0-9]+$/.test(id)
...
+  <p>
+    If you would like to pay with a cost center, consider creating a <a
```
Comment (merchako) [inline src/components/NoHouseholdIdModal.svelte:54]: Karl and I encourage contractions.
```
    If you'd like to pay with a cost center, consider creating a <a
```

File: src/components/NoHouseholdIdModal.svelte [hunk 4]
```diff
@@ -49,6 +50,12 @@ const isIdValid = (id: string): boolean => /^[0-9]+$/.test(id)
...
+      class="mdc-theme--primary"
+      href={POLICY_NEW_TEAM}>Team Policy</a
+    >
+  </p>
   <span class="header">Household ID<span class="required">*</span></span>
   <TextField placeholder={'1234567'} bind:value={householdId} /></Dialog.Alert
```
Comment (merchako) [inline src/components/NoHouseholdIdModal.svelte:60]: Better off with no placeholder text here.
```
  <TextField bind:value={householdId} /></Dialog.Alert
```
Article: 
[Placeholders in Form Fields Are Harmful](https://www.nngroup.com/articles/form-design-placeholders/) 
See #6. Users may mistake a placeholder for data that was automatically filled in.

File: src/components/NoHouseholdIdModal.svelte (hunk 2 above)
Comment (forevermatt) [reply src/components/NoHouseholdIdModal.svelte:52]: Minor issue, but the period should go before the ending `</p>` tag.

**Review by RussellChamp — APPROVED** [review]
code lgtm

**Pull request merged**